eclass/kernel-install.eclass

```diff
@@ -661,6 +661,18 @@ kernel-install_install_all() {
 			kernel-install_extract_from_uki initrd \
 				"${image_dir}"/uki.efi \
 				"${image_dir}"/initrd
+			kernel-install_extract_from_uki cmdline \
+				"${image_dir}"/uki.efi \
+				"${T}"/cmdline
+
+			elog
+			elog "The UKIs built-in kernel command line was set to:"
+			elog $(<"${T}"/cmdline)
+			elog "At boot this may be overridden by the firmware or bootloader"
+			elog "only if Secure Boot is disabled. If Secure Boot is enabled"
+			elog "then any kernel command line arguments parsed at boot are"
+			elog "ignored and the built-in kernel command line is used instead."
+			elog
```

Comment:
NAK. This is going to make a lot of noise on every single install of kernel.

Reply:
> NAK. This is going to make a lot of noise on every single install of kernel.

Specifically if we add `lockdown=integrity` then I feel it makes sense to alert the user that this will be part of the default cmdline. But I guess we can also put it on the wiki instead.

Reply:
The user will see this as a verbose generic messages instead of a warning that their system will not boot anymore.

Reply:
Alright, dropped this commit.